source/nodejs/adaptivecards/src/card-elements.ts

```diff
@@ -5708,9 +5709,23 @@ class OverflowAction extends Action {
                 contextMenu.items.add(menuItem);
             }
 
+            contextMenu.onClose = () => {
+                if (this.renderedElement) {
+                    this.renderedElement.ariaExpanded = "false";
```

Comment:
Sadly, Firefox [doesn't yet support](https://caniuse.com/mdn-api_element_arialabel) `aria` mixins, so you'll need to drop back to using `get/setProperty` like we do elsewhere. ☹️ 

e.g.

```suggestion
                    this.renderedElement.setProperty("aria-expanded", "false");
```

Reply:
I didn't know that - thanks for letting me know!